src/qt/mempoolstats.cpp

```diff
@@ -135,7 +135,8 @@ void MempoolStats::drawChart()
         allDataLabel->setFont(QFont(LABEL_FONT, LABEL_KV_SIZE, QFont::Light));
     }
 
-    static const QString checkbox_style_base = QStringLiteral("background-color:rgb(255,255,255);");
+    // Use transparent background to respect system theme (dark/light mode)
+    static const QString checkbox_style_base = QStringLiteral("background-color:transparent;");
```

Comment:
What happens if we simply delete the background-color?

```suggestion
    static const QString checkbox_style_base;
```

Reply:
still need the transparent background, otherwise we have a mismatch in background color for the checkboxes